.circleci/config.yml

```diff
@@ -415,9 +415,53 @@ jobs:
       - restore_cache:
           keys:
             - v4-front-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install libxtst6 for visual regression testing
+          command: |
```

Comment:
Curiosity, why did you need that ?
I feel like ca-certificates, wget or libstdc++6 are pretty far from image snapshoting

Reply:
Those are Puppeteer dependencies which is used under the hood of jsdom-screeshot
https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix
https://github.com/dferber90/jsdom-screenshot#how-it-works
I changed the job name accordingly, btw.

Reply:
thanks